### PR TITLE
Kuba for loop

### PR DESCRIPTION
**Pull request by KubaCZ007**
for loop

### CODE REVIEW

**Comment by KubaCZ007** [comment]
[Cykly (For Loop)](https://trello.com/c/gbhWeC9U/50-cykly-for-loop)

**Comment by KubaCZ007** [comment]
[Cykly (For Loop)](https://trello.com/c/gbhWeC9U/50-cykly-for-loop)

File: src/main/java/org/cakelook/for_Loop/Main.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.cakelook.for_Loop;
+
+public class Main {
+    public static void main(String[] args) {
+        int count = 100;
+        for (int i = 0; i < count; i++ ){
+            System.out.println(i);
+        }
+    }
+}
```
Comment (Kostahe) [inline src/main/java/org/cakelook/for_Loop/Main.java:10]: Insane promena count

File: src/main/java/org/cakelook/for_Loop/Main.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.cakelook.for_Loop;
+
+public class Main {
+    public static void main(String[] args) {
+        int count = 100;
```
Comment (Kostahe) [inline src/main/java/org/cakelook/for_Loop/Main.java:5]: Insane promena count

**Review by Kostahe — COMMENTED** [review]
Do you accept the mouse pay?

**Pull request merged**